.travis.yml

```diff
@@ -19,6 +19,10 @@ install:
   # Remove the (brew update) at that time.
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew install gpgme ; fi
 
+before_script:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then ; fi
```

Comment:
What does this do?

Reply:
Just a silly artifact when playing around :^)